client/overview/index.js

```diff
@@ -3,16 +3,19 @@
 /**
  * External dependencies
  */
+
 /**
  * Internal dependencies.
  */
 import Page from 'components/page';
 import { TestModeNotice, topics } from 'components/test-mode-notice';
+import AccountStatus from 'components/account-status';
 
 const OverviewPage = () => {
 	return (
 		<Page>
 			<TestModeNotice topic={ topics.overview } />
+			<AccountStatus { ...wcpaySettings } />
```

Comment:
It took me some time to realize this component is different from `client/account-status/index.js`. We need to come up with better naming while we have it displayed in two different places.

Reply:
`wcpaySettings` includes many params and if I understand correctly AccountStatus consumes only couple of them. In this case I'd suggest passing them explicitly to the component as props:
```js
<AccountStatus accountStatus={wcpaySettings.accountStatus} accountFees={wcpaySettings.accountFees} />
```

This will make migration to TS easier and give us more clarity which wcpaySettings props are consumed in status block.